index.html

```diff
@@ -81,7 +81,7 @@ <h1 class="personal-info-name side-padding">
                 <ul>
                     <li>Surfing</li>
                     <li>Snowbording</li>
-                    <li>Programming</li>
```

Comment:
I believe that you can leave `Programming` on this list

Reply:
sure